sir_cake/seller/views.py

```diff
@@ -36,7 +36,8 @@ def new_item(request):
     else:
         form = NewItemForm()
 
-    return render(request, 'seller/new_item.html', {'form': form})
+    return render(request, 'seller/new_item.html',
```

Comment:
Would it make sense to call `_get_url_name_for_navbar` in a context preprocessor?